mirage/fixtures/external-storage-services.ts

```diff
@@ -0,0 +1,124 @@
+// TODO: needs iconUri and authUri
+export default [
+    {
+        id: 'bitbucket',
+        name: 'Bitbucket',
+        readOnly: true,
+        supportsCopy: false,
+        supportsUserSpecificRootFolder: false,
+        supportsFileVersioning: false,
+        supportsBulkDownload: false,
```

Comment:
Well, these will be a bit of a pain to redo. A shame they didn't have this ready beforehand.